rebuild.sh

```diff
@@ -13,6 +13,20 @@ else
     echo "Shortcut change already confirmed"
 fi
 
+# Ensure TypeScript is new enough, offer to install with npm if not
+export PATH="./node_modules/.bin:$PATH"
+REQ_TSC_VER="3.8"
+CUR_TSC_VER=$(tsc --version 2>/dev/null || echo 0)
+CUR_TSC_VER=${CUR_TSC_VER##* }
+if [ "$(/bin/echo -e "${REQ_TSC_VER}\n${CUR_TSC_VER}" | sort -V | head -n1)" != "$REQ_TSC_VER" ]; then
```

Comment:
I think this will still always ask to install if the version is greater than `tsc 3.8`

Reply:
It is supposed to do the opposite, and that's how it works when I've tested it.

Could you run the script under `bash -x` and paste the output up until the point where it asks, please? Also give output from `tsc --version`.

Reply:
My bad, I just tested your branch and it seems to work fine using `@latest, @next, @rc, @beta` , etc.

Reply:
It looks like the oldest `3.8` is `3.8.3` - that seems to work fine too. Might want to update README.md#developers to have nodejs as a dependency for npm.

Reply:
The minimum version requirement of `3.8` was blindly copied from [README.md](https://github.com/pop-os/shell/blame/master_focal/README.md#L69) - I haven't verified that this information is correct. I could certainly update this to `3.8.3` if that is the *actual* minimum version requirement of course, but in that case, `README.md` should be updated as well. I feel that is outside the scope of this PR, though.

Reply:
I've now added a comment to `README.md` stating that `npm` will be used to satisfy the TypeScript dependency if necessary.